samples/msgext-ai-sentiment-analysis/nodejs/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "ME-ai-sentiment-analysis",
+    "version": "1.0.0",
+    "description": "Sentiment analysis using message extension",
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "index.js",
+    "scripts": {
+        "dev:teamsfx": "npm run start",
+        "start": "node ./index.js",
+        "watch": "nodemon ./index.js",
+        "build": "node build.js",
+        "lint": "eslint .",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
+    },
+    "dependencies": {
+        "@microsoft/microsoft-graph-client": "^3.0.4",
+        "@microsoft/teams-ai": "^0.1.0-preview.2",
+        "axios": "^0.21.0",
+        "botbuilder": "~4.18.0",
+        "botbuilder-dialogs": "^4.18.0",
+        "cors": "^2.8.5",
+        "dotenv": "^8.2.0",
+        "ejs": "^3.1.6",
+        "express": "^4.17.1",
+        "html-entities": "^1.3.1",
+        "isomorphic-fetch": "^3.0.0",
+        "memorystorage": "^0.12.0",
+        "node-schedule": "^2.1.0",
```

Comment:
Check if all of these packages are required.

Reply:
fixed